starter-code/index.js

```diff
@@ -1,11 +1,46 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor() {
+    this.items = [];
+    this.length = 0;
+  }
+  add(item) {
+    this.items.push(item);
+    this.items.sort();
```

Comment:
You're missing something from your sort function which is why some of the other tests are not passing 

```
 this.items.sort(function(a, b) {
      return a - b;
    });
```